cabal.project

```diff
@@ -9,30 +9,24 @@ constraints:
   hashable < 1.3,
   primitive < 0.7
 
-source-repository-package
-  type: git
-  location: http://github.com/well-typed/canonical-json
-  tag: ddfe3593b80b5ceb88842bb7a6f2268df75d2c2f
-
-source-repository-package
-  type: git
-  location: https://github.com/well-typed/cborg
-  tag: 80fbe0ee5e67a5622e2cb9eaa9d8594a2214322d
-  subdir: cborg
-
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
-  tag: 1121579803319f01612583fc445c50b0a0ce0424
+  tag: 31aafc8d5c090c4443a12d588c923badac929843
   subdir: binary
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
-  tag: 1121579803319f01612583fc445c50b0a0ce0424
+  tag: 31aafc8d5c090c4443a12d588c923badac929843
   subdir: cardano-crypto-class
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-prelude
-  tag: 4c48d34fd60991bb1b9168d293cfe31a975d858e
+  tag: b83730dccab14111d62144b048084eec7900d091
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/goblins
+  tag: 2d922d01289f926295f3b2d515138510b87768dc
```

Comment:
Ah, did I fail to add this? If people are building with `stack` and `cabal new-*` then we might want a CI check using `cardano-repo-tool` like the one I added to `cardano-ledger`.